css-highlight-api-1/Overview.bs

```diff
@@ -579,6 +579,42 @@ Range Updating and Invalidation</h3>
 	is not <a spec=dom for="StaticRange">valid</a>,
 	the user agent must ignore that [=range=].
 
+<h2 id="interactions">
+Interacting with Custom Highlights</h2>
+
+User interaction with [=custom highlights=] can be done via the
+ {{highlightsFromPoint}}(<var>x</var>, <var>y</var>, <var>options</var>) method.
+
+<pre class=idl>
+partial interface HighlightRegistry {
+  sequence&lt;Highlight> highlightsFromPoint(float x, float y, optional <span>HighlightsFromPointOptions</span> options = {});
+};
+
+dictionary <dfn dictionary>HighlightsFromPointOptions</dfn> {
+  sequence&lt;ShadowRoot> <dfn dict-member for="HighlightsFromPointOptions">shadowRoots</dfn> = [];
+};
+</pre>
+
+	The <dfn method for="HighlightRegistry">highlightsFromPoint(<var>x</var>, <var>y</var>, <var>options</var>)</dfn>
+	 method must return the result of running these steps:
+
+1. If there is no <a>viewport</a> associated with the document, return the empty sequence.
+1. If either argument is negative, <var>x</var> is greater
+    than the <a>viewport</a> width excluding the size of a rendered
+    scroll bar (if any), or <var>y</var> is greater than the
+    <a>viewport</a> height excluding the size of a rendered scroll bar
+    (if any), return the empty sequence.
+1. Otherwise, return a sequence of custom higlights ordered by [=priority=],
```

Comment:
The step should say something about where we're getting these highlights, e.g. "Return a sequence of [=custom highlights=] given by ordering the highlights contained in this {{HighlightRegistry}} by [=priority=], excluding the highlights without at least one range that satisfies the following constraints..."